### PR TITLE
feat(multi-env): "teamsfx env activate"

### PR DESCRIPTION
**Pull request by a1exwang**
- add teams env activate command and unit test
- display active mark on "teamsfx env list"

Screenshot from 2021-09-10 17-07-15

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10844614

### CODE REVIEW

**Review by chagong — APPROVED** [review]

File: packages/cli/src/helpParamGenerator.ts [hunk 1]
```diff
@@ -49,6 +49,7 @@ export class HelpParamGenerator {
     Stage.grantPermission,
     Stage.checkPermission,
     "validate",
+    Stage.switchEnv,
```
Comment (dooriya) [inline packages/cli/src/helpParamGenerator.ts:52]:
```suggestion
    Stage.activeEnv,
```

File: packages/cli/src/cmds/env.ts [hunk 2]
```diff
@@ -70,9 +73,56 @@ class EnvList extends YargsCommand {
...
+  public readonly command = `${this.commandHead}`;
+  public readonly description = "Activate an environments.";
+  public params: { [_: string]: Options } = {};
+
+  public builder(yargs: Argv): Argv<any> {
+    this.params = HelpParamGenerator.getYargsParamForHelp(Stage.switchEnv);
+    return yargs.version(false).options(this.params).demandOption(EnvNodeNoCreate.data.name!);
+  }
+
+  public async runCommand(args: { [argName: string]: string }): Promise<Result<null, FxError>> {
```
[the remark this answers is not in the document]
Comment (a1exwang) [reply packages/cli/src/cmds/env.ts:105]: Other commands like `provision`, `deploy` in CLI use `null`, so I think just keep consistent with the style in CLI.

[the remark this answers is not in the document]
Comment (qinezh) [reply packages/cli/src/cmds/env.ts:105]: ok, that would fine.

File: packages/cli/src/helpParamGenerator.ts (hunk 1 above)
Comment (a1exwang) [reply packages/cli/src/helpParamGenerator.ts:52]: Fixed

**Review by dooriya — APPROVED** [review]
:shipit:

**Review by qinezh — APPROVED** [review]

**Review by jayzhang — APPROVED** [review]

**Pull request merged**